public/css/main.css

```diff
@@ -100,18 +100,29 @@ textarea {
    header,
    main,
    footer {
-     margin-left: calc(50% - 490px);
-     padding: 0;
-     width: 980px;
-     margin-right: calc(50% - 490px);
+     max-width: 1000px;
+     margin: 0 auto;
    }
+
    header {
      border-bottom: solid #444 1px;
    }
+
    footer {
-     border-top: solid 1px #444;
+     position: relative;
    }
 
+   footer:after {
+     content: "";
+     position: absolute;
+     top: 0;
+     left: 0;
+     right: 0;
+     height: 1px;
+     background-color: #444;
+   }
+
+
```

Comment:
it looks like some merge conflicts got resolved the wrong way as this is older code...